io/net/virt-net/network_virtualization.py

```diff
@@ -587,6 +587,22 @@ def is_auto_failover_enabled(self):
                 return True
         return False
 
+    def enable_auto_failover(self):
+        """
+        Function to enable auto failover option
+        """
+        cmd = 'chhwres -r virtualio -m %s --rsubtype vnic \
+               -o s --id %s -s %s -a auto_priority_failover=1' \
+               % (self.server, self.lpar_id, self.slot_num)
+        try:
+            self.run_command(cmd)
+        except CommandFailed as cmdfail:
+            self.log.debug(str(cmdfail))
+            self.fail("lshwres operation failed ")
```

Comment:
Let us not fail from this function.
Please either return True or False, and take action from the calling code

Reply:
done